### ISSUE TITLE
I want to be able to specify my csv exports to use UTF-8 BOM

### ISSUE THREAD

**Issue by TuringLovesDeathMetal** [lightdash/lightdash#10214]
### Description

Right now, we allow users to export files as csv types. 

This is great, but for some languages (e.g. Japanese) exporting to CSV in UTF-8 actually causes issues with characters. 

Excel isn't very good at recognizing UTF-8, so, to fix this, you need to actually import the .csv to excel as UTF-8 BOM. 

There's a discussion here which makes it more clear what's going on: https://stackoverflow.com/questions/6002256/is-it-possible-to-force-excel-recognize-utf-8-csv-files-automatically

Basically, Lightdash should offer the option to users to import .csv files to excel using UTF-8 BOM (or maybe we just use this by default?)
